rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewAttachEvent.java

```diff
@@ -0,0 +1,59 @@
+package com.jakewharton.rxbinding.view;
+
+import android.content.Context;
+import android.support.annotation.IntDef;
+import android.support.annotation.NonNull;
+import android.view.View;
+
+/**
+ * A view attach event on a view.
+ * <p>
+ * <strong>Warning:</strong> Instances keep a strong reference to the view. Operators that
+ * cache instances have the potential to leak the associated {@link Context}.
+ */
+public final class ViewAttachEvent extends ViewEvent<View> {
+
+  public static final int ATTACH = 0;
+  public static final int DETACH = 1;
+
+  @IntDef({ATTACH, DETACH})
+  public @interface Kind {}
```

Comment:
All other events use an enum. If people want it changed that's a topic for a separate issue and should be done all at once. Switch this guy over.